modules/dcache/src/main/java/diskCacheV111/poolManager/PoolSelectionUnitV2.java

```diff
@@ -2749,8 +2750,16 @@ public String ac_psu_ls_netunits(Args args)
     @AffectsSetup
     public String ac_psu_remove_pool_$_1(Args args)
     {
-        removePool(args.argv(0));
-        return "";
+	if(Glob.isGlob(args.argv(0)))
+	{
+		Glob glob = new Glob(args.argv(0));	
+		List<String> names = getPools(glob.toPattern()).stream().map((Pool pool) -> pool.getName()).collect(Collectors.toList());
+		for(String name : names)
```

Comment:
please use curly bases  even with e single statement, e.g:

for (...) {
  removePool(name);
}